src/iop/lowpass.c

```diff
@@ -590,8 +590,6 @@ void gui_init(struct dt_iop_module_t *self)
   g->brightness = dt_bauhaus_slider_from_params(self, N_("brightness"));
   g->saturation = dt_bauhaus_slider_from_params(self, N_("saturation"));
 
-  dt_bauhaus_widget_set_label(g->brightness, NULL, NC_("lowpass", "brightness"));
```

Comment:
This is certainly a leftover from earlier refactoring. See abbf0c380b496f6c8f8dd3da1ac82f8b2d0563d7.

Reply:
Agreed !

Reply:
I guess I was trying to be careful not to break any existing translations that required a context, because introspection based translations at the moment don't support that yet. 

However, in this particular case I think we can fall back, as you suggest, to the contextless translation. After checking with the Hebrew translator, because the spreadsheet attached to https://github.com/darktable-org/darktable/pull/6648#issue-510602868 shows that's the only place where lowpass has a deviating translation and I don't know if that is intentional...

The reason this now breaks, btw, is that the interface of dt_bauhaus_widget_set_label has changed to expect untranslated strings and then translates them itself. However without switching to xgettext and defining an NQ_ equivalent to NC_ there's no clean way to pass context sensitive translateable strings. Just a workaround like 5adb238df3690eb737b3eeed14815aca28413434

Reply:
Ok, the context was added by my long time ago (August 2013)  in 345e8bc30. This was to have a specific translation for brightness in French... But at the moment it is translated as all other brightness instances. So probably something was changed and all this is OBE, let's merge this,